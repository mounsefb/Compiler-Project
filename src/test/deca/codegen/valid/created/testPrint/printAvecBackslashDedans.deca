// Description :
//    Test de l'affichage d'une chaîne de caractères avec un double backslash
//
// Résultats :
//    Hello World! je suis \\MrBackslash\\


{
    print("Hello World! je suis \\\\MrBackslash\\\\");
}
