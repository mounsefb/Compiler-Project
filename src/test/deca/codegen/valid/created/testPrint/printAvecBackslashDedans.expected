Hello World! je suis \\MrBackslash\\
